static/app/views/performance/mobile/appStarts/screenSummary/index.tsx

```diff
@@ -109,120 +109,130 @@ function ScreenSummary() {
+                  {
+                    unit: DurationUnit.MILLISECOND,
+                    allowZero: false,
+                    title:
+                      appStartType === COLD_START_TYPE
+                        ? t('Cold Start (%s)', PRIMARY_RELEASE_ALIAS)
+                        : t('Warm Start (%s)', PRIMARY_RELEASE_ALIAS),
+                    dataKey: `avg_if(span.duration,release,${primaryRelease})`,
+                  },
+                  {
+                    unit: DurationUnit.MILLISECOND,
+                    allowZero: false,
+                    title:
+                      appStartType === COLD_START_TYPE
+                        ? t('Cold Start (%s)', SECONDARY_RELEASE_ALIAS)
+                        : t('Warm Start (%s)', SECONDARY_RELEASE_ALIAS),
+                    dataKey: `avg_if(span.duration,release,${secondaryRelease})`,
+                  },
+                  {
+                    unit: 'percent_change',
+                    title: t('Change'),
+                    dataKey: `avg_compare(span.duration,release,${primaryRelease},${secondaryRelease})`,
+                  },
+                  {
+                    unit: 'count',
+                    title: t('Count'),
+                    dataKey: 'count()',
+                  },
+                ]}
+                referrer="api.starfish.mobile-startup-totals"
+              />
+            </HeaderContainer>
+            <ErrorBoundary mini>
+              <AppStartWidgets additionalFilters={[`transaction:${transactionName}`]} />
+            </ErrorBoundary>
+            <SamplesContainer>
+              <SamplesTables transactionName={transactionName} />
+            </SamplesContainer>
+            {spanGroup && spanOp && appStartType && (
+              <ScreenLoadSpanSamples
+                additionalFilters={{
+                  [SpanMetricsField.APP_START_TYPE]: appStartType,
+                  ...(deviceClass ? {[SpanMetricsField.DEVICE_CLASS]: deviceClass} : {}),
+                }}
+                groupId={spanGroup}
+                transactionName={transactionName}
+                spanDescription={spanDescription}
+                spanOp={spanOp}
+                onClose={() => {
+                  router.replace({
+                    pathname: router.location.pathname,
+                    query: omit(
+                      router.location.query,
+                      'spanGroup',
+                      'transactionMethod',
+                      'spanDescription',
+                      'spanOp'
+                    ),
+                  });
+                }}
+              />
+            )}
+          </Layout.Main>
+        </Layout.Body>
+      </PageAlertProvider>
+    </Layout.Page>
+  );
+}
+
+function PageWithProviders() {
+  const location = useLocation<Query>();
+
+  const {transaction} = location.query;
+
+  return (
+    <ModulePageProviders
+      title={[transaction, ROUTE_NAMES['app-startup']].join(' — ')}
```

Comment:
It looks like this `ROUTE_NAMES` constant is only really used by mobile modules. If you wanted to remove the constant and just define the title inline (i.e. `t(App Starts)`) then that removes another file from the old starfish path

Reply:
Ah good call, I can take a look separately